In [24]:
import pandas as pd
import seaborn as sn

In [4]:
df = pd.read_excel("./Banco de Dados Copos-BI.ods", engine="odf")

# Variavel resposta
## Preço Total
 Variavel do tipo **QUANTITATIVA**. A proposta aqui é observar como as zonas eleitorais e setores do TRE estão se adequando as propostas de sustentabilidade quanto a gestão de gastos públicos e diminuição de produção de lixo.

# Resumo da Variável resposta

In [39]:
df['Vl.Total'].describe()

count    188.000000
mean      22.467926
std       23.356196
min        1.190000
25%        8.650000
50%       13.160000
75%       31.680000
max      144.240000
Name: Vl.Total, dtype: float64

O valor máximo de **144.24** parece fora do padrão, talvez seja interessante analisar mais sobre em qual periodo e qual setor obteve tal gasto.

# Traduzindo
Levando em consideração as novas diretrizes em relação a gastos públicos, [PEC 241](https://www.politize.com.br/teto-de-gastos-publicos-infografico/), e a crescente importância com questões ambientais, foi sugerido pela AGE que se extinguisse a utlização de copos descartáveis pelos setores do TRE e diminuissem ao máximo sua utilização nas zonas eleitorais. Ao observar que a média de gasto mensais totais com copos descatáveis é de aproximadamente R$ 22.46, faz-se necessário averiguar se essas unidades estão de fato diminuindo o consumo de copos descatáveis e porque tivermos casos tao destoantes como na seção de conservação predial. 

In [52]:
df[['Unidade Gestora','Unidade','Vl.Total']][df['Vl.Total']==df['Vl.Total'].max()]

Unidade Gestora                       Unidade  Vl.Total
179             NaN  SEÇÃO DE CONSERVAÇÃO PREDIAL    144.24

# Variaveis explicativas

|**Variável**|**Tipo**|
|----------|:-------------:|
|Unidade Gestora   |QUALITATIVA   |    
|Mês    |  QUALITATIVA |  
|Quantidade    |  QUANTITATIVA |
 